packages/clerk-js/src/core/clerk.ts

```diff
@@ -1137,16 +1137,23 @@ export default class Clerk implements ClerkInterface {
       clerkMissingSignInUrlAsSatellite();
     }
 
-    return this.#instanceType === 'development';
+    // Attempt to sync if user is logged out or session has expired
+    const cookieHandler = createCookieHandler();
+    return (
+      this.#instanceType === 'development' &&
```

Comment:
@panteliselef couldn't these checks be simplified if we moved the `this.#instanceType === 'development'` logic [here](https://github.com/clerkinc/javascript/blob/791a7e075bad7a3e75b614ab24907f62197fcee5/packages/clerk-js/src/core/clerk.ts#L1132)?

(same in prod case below)

Reply:
I don't think this would help, because these methods are also used in [here ](https://github.com/clerkinc/javascript/blob/c72aa607a9b5fef69953abbfec9c47b37f17d5eb/packages/clerk-js/src/core/clerk.ts#L1173)